Source/WebCore/platform/graphics/FontCascade.cpp

```diff
@@ -1254,10 +1254,15 @@ bool FontCascade::canReceiveTextEmphasis(char32_t c)
         return false;
 
     // Additional word-separator characters listed in CSS Text Level 3 Editor's Draft 3 November 2010.
+    // https://www.w3.org/TR/css-text-3/#word-separator
     if (c == ethiopicWordspace || c == aegeanWordSeparatorLine || c == aegeanWordSeparatorDot
         || c == ugariticWordDivider || c == tibetanMarkIntersyllabicTsheg || c == tibetanMarkDelimiterTshegBstar)
         return false;
 
+    // Punctuations
+    if (U_GET_GC_MASK(c) & U_GC_P_MASK)
+        return false;
```

Comment:
I realized this can be combined with the checks above and should not be a separate if statement. Just or in `U_GC_P_MASK` with the other masks in the expression at the top of the function, which is already checking multiple categories. Adding punctuation doesn't need to be separate lines of code with a separate comment. Adding a comment would be OK, but one just about punctuation doesn't make a ton of sense.
```
... U_GC_CF_MASK | U_GC_P_MASK))
```

Reply:
Done. :-)